doc/user/setup.rst

```diff
@@ -83,6 +71,13 @@ This file has several parts. Here is an example:
 
 Breaking this file down:
 
+::
+
+  bgpd=yes
```

Comment:
Three spaces pls. Also, for future reference you can write blocks like this as:
```
Breaking this file down::

   bgpd = yes
```
This is syntax sugar, Sphinx will render the `::` as a single `:` and treat the indented block as a code block.

Reply:
Took the liberty of adding that space character and merging.

btw, for future reference, shell doesn't accept spaces around the `=` :laughing: